epub33/overview/index.html

```diff
@@ -477,10 +555,10 @@ <h1>Accessibility</h1>
 				this requirement. This section reviews these features, detailing some established best practices for
 				ensuring that EPUB Publications are accessible where applicable.</p>
 
-			<p>EPUB 3 also includes an Accessibility specification [[EPUB-A11Y-10]] that leverages the extensive work
-				done to make Web content accessible in [[WCAG20]]. The specification defines requirements to produce
+			<p>EPUB 3.3 also includes an Accessibility specification [[EPUB-A11Y-11]] that leverages the extensive work
```

Comment:
The accessibility specification isn't directly tied to 3.3, but that's what it sounds like here. I would leave off the minor revision number, as in the original, or see if there's some other way to phrase that it applies to EPUB 3.3 without making it appear to be a product of this revision.

Reply:
I have removed the minor revision number. I am not sure there is any change that can be done in the section...